ci_framework/roles/devscripts/README.md

```diff
@@ -12,14 +12,16 @@ network configuration, repository setup and libvirt networks.
 ## Parameters
 
 * `cifmw_devscripts_artifacts_dir` (str) path to the directory to store the role artifacts.
-* `cifmw_devscripts_ci_token` (str) oAuth token required for accessing console.redhat.com.
-* `cifmw_devscripts_ci_token_file` (str) oAuth token file required for accessing console.redhat.com.
-   We can use `cifmw_devscripts_ci_token` or `cifmw_devscripts_ci_token_file` for passing OAuth token.
+* `cifmw_devscripts_ci_token` (str) oAuth token required for accessing
+  [openshift-console](https://console-openshift-console.apps.ci.l2s4.p1.openshiftapps.com/).
+* `cifmw_devscripts_ci_token_file` (str) oAuth token file required for accessing
+  [openshift-console](https://console-openshift-console.apps.ci.l2s4.p1.openshiftapps.com/).
+  We can use `cifmw_devscripts_ci_token` or `cifmw_devscripts_ci_token_file` for passing OAuth token.
 * `cifmw_devscripts_config_overrides` (dict) key/value pairs to be used for overriding the default
   configuration. Refer [section](#supported-keys-in-cifmw_devscripts_config_overrides) for more information.
-* `cifmw_devscripts_crb_repo` (str) Repo URL of code ready builder.
 * `cifmw_devscripts_dry_run` (bool) If enabled, the workflow is evaluated.
 * `cifmw_devscripts_make_target` (str) Optional, the target to be used with dev-scripts.
+* `cifmw_devscripts_restart_virtproxyd` (bool) Optional, if libvirt's virtproxyd service should be restarted.
```

Comment:
Would be good to add an info that it's [enabled by default](https://github.com/openstack-k8s-operators/ci-framework/pull/659/files#diff-29b79ce24c92db9953622004237916ce6b38c6fe2f1ee05a9a1607e7c5b7081dR52)

Reply:
Thank you.

Yes, mentioning it does make sense.